tardis/styles/twentytwentyfour.json

```diff
@@ -0,0 +1,7 @@
+{
+	"$schema": "https://schemas.wp.org/wp/6.5/theme.json",
+	"version": 2,
+	"title": "TwentyTwentyFour",
```

Comment:
Not a blocker for this but the title might look bad on the UI if we write them like this